### PR TITLE
Snowplow integration

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/faqs/yes-you-can-faq**](https://docs-g4b7dufqp.avo-dev.app/faqs/yes-you-can-faq) | [**/workspace/integrations**](https://docs-g4b7dufqp.avo-dev.app/workspace/integrations) |
|:-:|:-:|
|<a href="https://docs-g4b7dufqp.avo-dev.app/faqs/yes-you-can-faq">Screenshot of /faqs/yes-you-can-faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/wbyJ9LN6mkXOmy8F8BJRjiP3juC8MhkNTVrMTV3E1k0G1ZwP7z7d8w5tbfcv_3K5Ihp3mVUDNP5u5s__Nmxd_g.png">(view full size)</a> |<a href="https://docs-g4b7dufqp.avo-dev.app/workspace/integrations">Screenshot of /workspace/integrations</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/D_EH0NBPOEN4E0SGSULP5O3FBh4-_S-9jRrE4iwVS2UenpTq6Uzb12bPdvQ8uMW2kFLXVPoHgJz_4t5SqSrUIA.png">(view full size)</a> |



| [**…acking-plan/publishing**](https://docs-g4b7dufqp.avo-dev.app/workspace/tracking-plan/publishing) |
|:-:|
|<a href="https://docs-g4b7dufqp.avo-dev.app/workspace/tracking-plan/publishing">Screenshot of /workspace/tracking-plan/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/O3TL5YQT9g9xdYgn1pDYZH5PUUK_GrkqBtVXpfnQjSy6S17_zLH6vNdStClvVeuenY0IsUVQyEBi3UJlRVKGGA.png">(view full size)</a> |







Commit a54217b43d080f18a8d3dd32950c9ab86cdb7948 (https://docs-g4b7dufqp.avo-dev.app).

File: pages/workspace/tracking-plan/publishing.mdx [hunk 1]
```diff
@@ -203,6 +204,34 @@ Avo will override all events in the mParticle Tracking Plan that are being impor
...
+
+The [Snowplow integration](https://docs.snowplowanalytics.com/docs/understanding-tracking-design/managing-data-structures/) enables you to publish your Avo Tracking Plan into Snowplow. You can configure the integration to publish whenever a branch is merged or with a push of a button.
+
+### Configure the Snowplow integration
+
+To enable the direct integration with Snowplow Data Structures, you need to configure it with the following parameters from Snowplow.
+
+- **Snowplow Organization ID**: Your organization ID issued by Snowplow
+- **Snowplow Vendor Name**: The vendor name for the data structures published by Avo, usually your domain in revers domain notation
+- **Snowplow Access Token**: The API Token provided by Snowplow for authentication
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:225]: Can we link to some Snowplow docs here for how to find these things?

File: pages/workspace/tracking-plan/publishing.mdx [hunk 2]
```diff
@@ -203,6 +204,34 @@ Avo will override all events in the mParticle Tracking Plan that are being impor
...
+
+### Publishing to Snowplow
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:227]: Should we add something about how we handle schema versioning?

**Comment by hugihlynsson** [comment]
@alex-tpom6oh what's the status on this?

**Comment by alex-tpom6oh** [comment]
@hugihlynsson this is under a feature flag now

**Pull request closed without merging**